model/k8s/registry.yaml

```diff
@@ -232,3 +232,69 @@ groups:
               value: 'terminating'
               brief: "Terminating namespace phase as described by [K8s API](https://pkg.go.dev/k8s.io/api@v0.31.3/core/v1#NamespacePhase)"
               stability: experimental
+      - id: k8s.container.status.state
+        stability: experimental
+        brief: >
+          The state of the container.
+        type:
+          members:
+            - id: terminated
+              value: 'terminated'
+              brief: 'The container has terminated.'
+              stability: experimental
+            - id: running
+              value: 'running'
+              brief: 'The container is running.'
+              stability: experimental
+            - id: waiting
+              value: 'waiting'
+              brief: 'The container is waiting.'
+              stability: experimental
+        examples: ['terminated', 'running', 'waiting']
+      - id: k8s.container.status.reason
+        type:
+          members:
+            - id: container_creating
```

Comment:
I don't have any objection to snake case, just want to make sure we want to stick with this convention going forward for multi word attribute values?

Reply:
Maybe @open-telemetry/specs-semconv-approvers could provide a direction on this?

Reply:
snake_case is the convention used for enum id 👍

btw, in case it's not clear, enum ids are only used for code generated constant names (they won't appear in the captured telemetry itself)